### PR TITLE
All addresses text is not updated when changing accounts

### LINKED ISSUE #19374: "All addresses" text is not updated after unselecting some addresses for permissions
**Steps:** 

1. Open Addresses for permissions screen
2. Unselect one of the addresses
3. Confirm changes
4. Pay attention at text under 'Join as a <role>'

**Expected result:** it should be updated according to the number of selected addresses

https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=15354-439774&mode=design&t=Vo7w6zOdm93ljfW0-0

Communities for Mobile – Figma 2024-03-22 19-10-55

**Actual result:**

https://github.com/status-im/status-mobile/assets/97245802/eefce54f-0ecd-4f13-ac89-2214074d8c29

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #19374 

### Summary

Fixes an issue with the text that indicated the number of selected addresses in Edit shared addresses screen and Request to join screen

status: ready

### CODE REVIEW

**Review by seanstrom — APPROVED** [review]
Nice stuff! ✅

File: src/status_im/contexts/communities/actions/accounts_selection/view.cljs [hunk 1]
```diff
@@ -20,6 +21,16 @@
         has-permissions? (rf/sub [:communities/has-permissions? id])
         airdrop-account (rf/sub [:communities/airdrop-account id])
         revealed-accounts (rf/sub [:communities/accounts-to-reveal id])
+        revealed-accounts-count (count revealed-accounts)
+        wallet-accounts-count (count (rf/sub [:wallet/accounts-without-watched-accounts]))
+        addresses-shared-text (condp = revealed-accounts-count
```
Comment (flexsurfer) [inline src/status_im/contexts/communities/actions/accounts_selection/view.cljs]: it's better to implement it similar to 
```clojure
"community-members": {
        "one": "{{count}} member",
        "other": "{{count}} members"
    },
    ```

**Review by flexsurfer — CHANGES_REQUESTED** [review]

**Review by jo-mut — APPROVED** [review]

File: src/status_im/contexts/communities/actions/accounts_selection/view.cljs (hunk 1 above)
Comment (ibrkhalil) [reply src/status_im/contexts/communities/actions/accounts_selection/view.cljs]: Done

**Review by flexsurfer — APPROVED** [review]

**Comment by qoqobolo** [comment]
Thanks a lot for the fix @ibrkhalil, PR can be merged.

**Pull request merged**